components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -251,7 +251,7 @@ ConversationDriver::GetVisibleConversationHistory() {
   // Remove conversations that are meant to be hidden from the user
   std::vector<ai_chat::mojom::ConversationTurnPtr> list;
   for (const auto& turn : GetConversationHistory()) {
-    if (turn.visibility != ConversationTurnVisibility::HIDDEN) {
+    if (turn->visibility != ConversationTurnVisibility::HIDDEN) {
```

Comment:
while you're changing the usage to use the `mojo::StructPtr<T>` variant for ConversationTurn everywhere, do you think we should also remove visibility? if I recall correctly, we don't use it anywhere.